### ISSUE TITLE
[ADE Support] Collaborate and share knowledge with MHV Accessibility specialists on setting up a shared service team

### ISSUE THREAD

**Issue by lakper** [department-of-veterans-affairs/va.gov-team#131622]
## Description of the Work
The MHV team reached out to ask for more information and support on how the ADE team is structured, to enable the MHV accessibility specialists to transition to a shared support operating model. The intent is to help the team organize to be efficient and provide support to as many team as possible within the MHV program. There is a recurring call set up every two weeks to discuss items and finalize next steps. 

## About the Team 

- **Team name**: MHV
- **Product name:** MHV
- **Product manager:**  
- **Slack Channel:** [#mhv-accessibility](https://dsva.slack.com/archives/C05QG2H510T)

This ticket will track tasks related to this work.
